apps/vs-code-designer/src/app/commands/pickFuncProcess.ts

```diff
@@ -35,17 +36,16 @@ export async function pickFuncProcess(context: IActionContext, debugConfig: vsco
   await callWithTelemetryAndErrorHandling(autoStartAzuriteSetting, async (actionContext: IActionContext) => {
     await runWithDurationTelemetry(actionContext, autoStartAzuriteSetting, async () => {
       await activateAzurite(context);
+      await verifyLocalConnectionKeys(context);
```

Comment:
NIT: can you take add another `runWithDurationTelemetry` and put the `verifyLocalConnectionKeys`. The `runWithDurationTelemetry` logs in the telemetry the time and if there is any error from whatever is getting executed within it. Or can you update the second param of the function, which is the prefix so we know its logging both of them?